apps.awesim.org/apps/activejobs/initializers/filter.rb

```diff
@@ -4,5 +4,5 @@
   f.title = "Your Group's Jobs (#{group})"
   f.filter_id = "group"
   # N.B. Need to use :egroup here for now. My Oodsupport group name is 'appl' but job 'Account_Name' is 'PZS0002'
-  f.filter_block = Proc.new { |job| job.native[:egroup] == group }
+  f.filter_block = Proc.new { |job| job.native[:egroup] == group || job.accounting_id == (group.downcase || group) || job.native[:account] == (group.downcase || group) }
```

Comment:
I'm not sure I get the use of `||` here.  Seems like it'll always use the `group.downcase` or at least `group.downcase` will never evaluate to nil using just group.

Reply:
I noticed some jobs `accounting_id` and `native.account` have a mixture of uppercase and lowercase group names.

`PZS1000` vs `pzs0715`

Reply:
Yea but I mean, are you trying to express `job.accounting_id == group.downcase ||  job.accounting_id == group`?

I _think_ `job.accounting_id == (group.downcase || group)` evaluates to `job.accounting_id == group.downcase` because it's using `||` for the _initialization_ of the right hand side.

I had to actually look this up and test it out, because I'm also legitimately wondering if ruby is this slick.  But it doesn't look like it. Unless `group.downcase` evaluates to nil or false, we'll never hit `group` in those expressions (and I'm not sure that it could).
```ruby
irb(main):001:0> "right" == ("left" || "right" )
=> false
```